---
title: "Week 2 Inclass"
permalink: /docs/w2inclass/
last_modified_at: Feb.20.2019
toc: true
---

## Friday In Class: Making with Mushrooms
**Think it |  Shape it |  Grow it |  Build it**

Recall from the safety training that the UTL space (our undergraduate learning labspace) requires that 
shoes must cover the entire foot and pants must cover the ankle. Please leave food and drinks on the cart outside.

<figure>
<a href="/assets/images/w8inclass_Making_with_Mushrooms.png"><img src="/assets/images/w8inclass_Making_with_Mushrooms.png"></a>
</figure>
**Figure-1:** Mycelium objects made by Bioe.80 students.

### Introduction:
Welcome to the Making with Mushrooms activity!
Our goal is to help you discover and get you inspired by the potential of living matter asa material.
In class goal: Make your own mycelium material object.
To complete this activity you will first make your own mycelium material.
Finally, you are asked to document and share your images as part of your PSET-8.


### Make your own mycelium material object:
If you would like a video tutorial please visit the MycoWorks Website [MycoWorks](https://www.youtube.com/watch?reload=9&v=c6nurN-Hii8). 
TAs will be available during the activity (Shriram 112 and 132) session to help you with making your mycelium material object. You can also explore theseInstructables on making mycelium objects [making mycelial forms](https://www.instructables.com/id/Mycelial-Forms/), [making mycelium](https://www.instructables.com/id/Making-Mycelium/), and making mycelium molds [making mycelium molds](https://www.instructables.com/id/Mycelium-Molds/).


### Materials
please check to see if you have all these items at your table:
2 large bowls, small bags/plastic wrap, cups, mushroom material, sharpie/tape
Molds are available

### Procedure

#### Set up 
(suggested time: 10 min)

1. Put on gloves!

2. Spray and wipe down your forms/molds.

3. Break the antlers off, if there are any, to get to the spongy layer. Put theantlers in one bowl and the spongy material in the other.

4. Squeeze out the juice from the spongy material into the antler bowl in orderto get a damp but not wet material, leaving it in its original bowl. This will beyour final material. Your damp material should have no excess water (i.e. kindof like wet sand).

#### Build 
(suggested time: 15 min)

5. Think of something you would like to build, perhaps the idea you came inwith or the object or part you wanted to mold.

6. Shape the material! Here are three ways to do so:
  a.With your hands!
  b.Around an object!
  c.In a baggy or cup!
  
7. Seal it! You can either put it in a plastic bag, plastic wrap, or cup.

*Remember, you want to LOOSELY SEAL your bag to allow for gas exchange so that the material can harden.*

#### Finish 
(suggested time: 10 min)

8. Label the bag/cup and ask your TA where to place it.

9. As a team, clean up your space completely. leaving the space better than you found it. 
To do so:

a. Take all biomaterials and anything that touched biomaterial (e.g. trash bag coverings on the tables in 112) and place them in the trash.

b. Wipe down your countertops with ethanol.

c. Please clean your bowls in the sink after dumping the mushroom wastematerial in the trash bins. Please give your bowl to a TA afterwards forstorage.


####  Guiding questions to think about as you make

These questions will be part of your PSET.

**Q.1** What is actually going on here? Describe in broad terms what processesunique to living matter are making this possible and the potential/limitation ofmaking with mushrooms.

**Q.2** How is this similar or different to other additive manufacturing processes suchas 3D printing? (detail out similarities/differences...feel free to draw)

**Q.3** Looking forward, how do we go from creating matter with an inert function(e.g. a chair) to creating matter with biological function (i.e. growing a finger, aheart, etc)? What new considerations need to be made?

#### Additional Resources:
1.MycoWorksa https://www.youtube.com/watch?v=VWQznqpy3Ssb 
https://www.youtube.com/watch?time_continue=1&v=3Dg6IleHm-k2

Bolt Threads: https://boltthreads.com
[link](https://www.forbes.com/sites/amyfeldman/2018/04/16/synthetic-spider-silk-maker-bolt-threads-debuts-new-bio-material-leather-made-from-mushroom-roots/#4eda3aea1837), [link](https://www.stellamccartney.com/experience/us/the-next-chapter-with-bolt-threads-mylo-material-rooted-in-nature/)

3D Printing Mycelium Materials: [link](https://www.wired.com/2013/12/mushroom-chair/),
 [link](http://www.ericklarenbeek.com)




## How do design, build, and test tools work to support bioengineering?
___________________________________________________

## Friday In Class GOOP
### GOOP 1: The Expression StatementIn 

this module we’ll learn about and execute the biomolecular equivalent of the ‘print’ statementby producing green fluorescent protein, or GFP. GFP and other fluorescent proteins are oftenused as an output to indicate the state of a genetic circuit, report on a sensor, or to identify cellscontaining a specific genetic payload. When it is produced, GFP is a visible yellow color (underambient light). When illuminated with light at 485 nm (blue), GFP produces fluorescent light at 510 nm (green).


<figure>
<a href="/assets/images/w2ic_GOOP_fig1.png"><img src="/assets/images/w2ic_GOOP_fig1.png"></a>
</figure>


**Figure-1:** GOOP tubes with and without the addition of expression statement (producing greenfluorescent protein) 

### Safety
There are no special safety concerns for this module. Recall that GOOP both cannot replicate,and is based on a BSL-1 organism. The green fluorescent protein is not known to be hazardousto health or the environment.

### Unpacking the Central Dogma

In the Preclass introduction, we discussed the central dogma of biology. Here, we explain howa particular gene ends up as a functional protein both inside a cell, or inside tubes. When aspecific gene is transcribed and translated, we say that is is **expressed**.

First, our gene must be transcribed. For transcription to occur, the gene must be inside atranscription unit. A transcription unit is composed of two special DNA sequences, a **promoterand** a **terminator**.

<figure>
<a href="/assets/images/w2ic_fig2.png"><img src="/assets/images/w2ic_fig2.png"></a>
</figure>

During transcription, RNA polymerase recognises the promoter DNA sequence and binds to it,beginning transcription at the transcription start site, just downstream of the promoter sequence.It then transcribes DNA to RNA until it reaches the terminator sequence. At the terminator, theRNA polymerase releases the DNA and the new RNA transcript.

Now the RNA transcript must be translated to a protein. The ribosome binds to a specificsequence on the RNA transcript, the **ribosome binding site**, and then searches along thetranscript until it finds the beginning of the gene sequence. The gene sequence begins with a **start codon**. The ribosome then translates codons into amino acids until it reaches a **stop codon**.

<figure>
<a href="/assets/images/w2ic_fig3.png"><img src="/assets/images/w2ic_fig3.png"></a>
</figure>

This combination of promoter, ribosome binding site, gene sequence, and terminator gives usour transcription unit and a complete construct with which to express a gene. We can alsoconsider this transcription unit a simple genetic **device**. There are a number of differentpromoters, ribosome binding sites, and terminators. Each behaves differently: differentpromoters may only be transcribed under certain conditions, or at different levels; ribosomebinding sites may be more or less attractive to the ribosome, changing the level of translation,and terminators can be ‘leaky’, only terminating transcription some of the time. Together withthe enormous diversity of genes available across the tree of life, these components make up thesimplest ‘parts’ we can use to build complex circuits.

### Expression Device (the ‘print’ statement)

For our expression statement, we’ll need the following:
- **p70a** a promoter that is active by default in GOOP, and initiates transcription at a highlevel.
- **UTR1** a sequence containing an efficient ribosome binding site.
- **deGFP** the coding sequence, or gene, for a green fluorescent protein, optimized for expression in GOOP.
- **T500** a strong (non-leaky) transcriptional terminator.

### Visualizing GFP with PyMol

While your GOOP is running, check out the structure of the GFP protein encoded by the DNA you used. 

For more information about GFP check out: [PDB Link](http://pdb101.rcsb.org/motm/42)

 1. Open PyMol on your laptop. As part of the pre-class assignment, you should have downloaded PyMol from this [Pymol Link]( https://pymol.org/2/)
 
 2. Type in the command “fetch 1gfl” in PyMol. This command downloads the structure ofGreen Fluorescent Protein (GFP) from the Protein Data Bank (PDB)
 
3. Check out the website where the GFP structure is housed:https://www.rcsb.org/structure/1gfl
 
4. Go back to PyMol and take a look at the GFP structure.  Click Hide everything and ‘Show cartoon’, under the menus “H” and “S”  listed near the structure name on the toolbar on the right. The GFP structure is a *beta barrel*. This means it is made up of abarrel formed by *beta strands*, a known type of secondary structure proteins adopt.

5. Click on   <Mouse → 1 button viewing> to enable scrolling.

6. Click on   <Display → Sequence On> to see the amino acid sequence for GFP.

7. Select residues 64-66 SYG and color these residues red. This is the *chromophore* of GFP, the portion of the molecule responsible for the fluorescence.

8. Render the SYG portion of the protein as ‘sticks’. You should now be able to see the cyclized ring of the GFP chromophore. This cyclization, as shown below, is part of the *maturation* process of the protein -- after maturation, the protein is able to fluoresce.

<figure>
<a href="/assets/images/w2ic_gfp_cyclization.png"><img src="/assets/images/w2ic_gfp_cyclization.png"></a>
</figure>

[source](http://www.cryst.bbk.ac.uk/PPS2/projects/jonda/chromoph.htm)


9. Take a screenshot of your highlighted protein structure for the problem set 2.


### GOOP Experiment

To set up our prototype experiment, we will our GFP device in a GOOP tube, following theinstructions in the kit introduction. We will also set up a *negative control*: a reaction tube withno DNA at all. We do this so that we can observe what GOOP by itself looks like, and compareit to our prototyped device.

In your kit you have:

- 2x GOOP reaction tubes labeled (G)
- 1x Module 1 DNA parts kit  labeled (D)
- 1x Pure water  labeled (W)
- 2x Transfer pipettes

1. Set up two tubes following experiments (additional instructions from the Pre-class):

<table>
  <tr>
    <td>Tube</td>
    <td>1</td>
    <td>2</td>
  </tr>
  <tr>
    <td>What you will add</td>
    <td>Water (negative control) </td>
    <td>DNA: p70a-UTR1-deGFP-T</td>
  </tr>
</table>

2. Note the time and let the reactions run. Take a photo of your tubes (at time zero). You shouldbegin seeing a color change after about 6 hours; after 12 hours there should be cleardifferences.

3. Take a photo of your GOOP tubes at the end of the experiment for the associated problemset. If you don’t see a difference between the tubes, don’t worry, but still take a photo. Note the time that you take your final photo (the endpoint of the experiment).


In today’s activity, you:
- Learned about the core genetic components necessary to express a gene
- Expressed your first gene in GOOP, producing green fluorescent protein
- Visualized the structure of the protein using PyMol.

By switching out the gene of our simple expression device, we can produce any protein from across the tree of life (although complications can arise). For example, by building an expression device with the gene you submitted to the Free Genesproject, you could produce the protein it encodes and test its function. Moreover, by changingother parts of the expression device (such as the promoter) you can control other aspects ofexpression, and by combining these simple devices we can build more complex circuits.

#### Additional Resources:

**Fluorescent Proteins:**
1. Fluorescent Proteins and the Story Behind GFP (by Roger Tsien ibiology) [link](https://www.ibiology.org/talks/fluorescent-proteins/)

2. Introduction to Fluorescent Proteins [link](https://www.microscopyu.com/techniques/fluorescence/introduction-to-fluorescent-proteins)

3. Interactive graph describing Fluorescent protein properties [link](http://www.fpvis.org/FP.html)

**Basic Parts:**

1. Adventures in Synthetic Biology - [comic book](http://web.mit.edu/endy/www/scraps/comic/AiSB.vol1.pdf)

2. iGEM Parts Expression [parts](http://parts.igem.org/Main_Page)

3. Expression statement DNA sequence designs [sequence](https://benchling.com/acjs/f_/MGZTmIMA-module-1/?sort=name&section=inventory&filter=archivePurposes%3ANOT_ARCHIVED)
