---
title: "Week 5 Preclass - Evolution "
permalink: /docs/w5preclass/
last_modified_at: Feb.20.2019
2018: Week 7
toc: true
---

## Evolution as an algorithm; Evolution as a service
_______________________________________________________________________

Please read and consider the below before start of each class.
The questions given are only study questions not homework to be graded.
Talk about it all with your classmates, friends, or TAs, as you like.
_______________________________________________________________________

## Preclass for Monday

## Introduction: Evolving Better Biology

## From Natural Algorithm to Engineering Service

This week you will examine the role of evolution from a Bioengineering perspective. 
After considering the following example and the material throughout this week you will be able to: 

**Goal-1:** Describe evolution as an algorithm that follows a set of rules.  

**Goal-2:** Utilize evolution as an algorithm. *“Evolution, at your service!”*:  
This robust algorithm is a service that can be deployed in diverse contexts. 

**Goal-3:** Describe the limitations when used as a service, and begin to describe what evolution 
could look like in the future *(“Evolution 2.0”)*.


**The Core concept for this week includes:** Evolution: Population, Diversity, Fitness, 
Selection (Pressure), Mechanics (Random Mutations), and Algorithm vs Service.

Before we begin briefly explore these: [link](https://www.nature.com/scitable/knowledge/evolution-13228138) 
and [link-2](https://www.nature.com/scitable/knowledge/library/mutations-are-the-raw-materials-of-evolution-17395346)
to better appreciate these examples.


## Example-1: Why do you need to get a flu shot every year?

To begin, let us consider this past flu season. (Which was considered as the worst flu season since 2009-2010. 
See Figure-1.) 

Following is from an [article](https://www.theatlantic.com/health/archive/2018/01/the-perfect-storm-behind-this-years-nasty-flu-season/550469/) 
published in January 2018: 

“Every winter brings cautionary tales that the flu—just the regular old flu—can kill. And the cautionary tales this year are hard to beat. Twenty-one-year-old Kyler Baughman, for example, a fitness buff who liked to show off his six-pack, recently died a few days after getting a runny nose. [ref](https://www.denverpost.com/2018/01/11/kyler-baughman-pennsylvania-personal-trainer-dies-of-flu/)


According to the numbers, this year’s flu season is in fact worse than usual. It got started early, and it’s been more severe. Twenty kids have died of the flu since October. And in the week ending January 6, 22.7 out of every 100,000 hospitalizations in the U.S. were for flu—twice the number of the previous week.” 


**ADD Figure-1: The flu season of 2017-2018 is the worst season since 2009-2010**


Every year, we are reminded to go get a flu shot. 

**Q.1. If you got a flu vaccine last year, why do you still have to get a flu shot again this year?**

**Q.2. What are the two populations that are interacting in the above story and graph?**

**Q.3. How do new flu strains arise every year? Where do they come from?**

**Q.4. What causes the flu virus to change?**

Briefly examine the two following links:

1. [Why You Need the Flu Shot Every Year](https://www.nytimes.com/2017/12/12/smarter-living/why-you-need-the-flu-shot-every-year.html)
Let’s highlight to the following from above link: “Tiny mutations help the virus evade detection, and as it evades detection, it’s more successful at multiplying”.

2. [How the Flu Virus Can Change](https://www.cdc.gov/flu/about/viruses/change.htm)


An **algorithm** can be defined as:  
“A process or set of rules to be followed in calculations or other problem-solving operations, especially by a computer.” (via Google) 

Given this definition, think about evolution in this new context.  

**Q.5. If evolution is an algorithm, what problem is being solved, and by whom(/what)? What are the rules?**

Extracting out the idea of Evolution from this example evolution is the results of two intertwined processes: Random Mutations beget genetic variation; non-random selection to improve “fitness”. The end result may look non-random, but the mechanism that enables evolution and natural selection to occur is entirely random. 


This example of flu virus highlights the algorithm of evolution acting on a *timescale* that is easy for us to understand.


## Example-2: Ancient virus genomes reveal complex evolution of Hepatitis B

In a fascinating [research article](https://www.biorxiv.org/content/early/2018/05/06/315531.full.pdf+html) see also ([article](https://www.nytimes.com/2018/05/09/science/ancient-dna-bones-hepatitis.html)), “Scientists reported on Wednesday that they have recovered DNA from the oldest viruses known to have infected humans — and have succeeded in resurrecting some of them in the laboratory. The viruses were all strains of hepatitis B. Two teams of researchers independently discovered its DNA in 15 ancient skeletons, the oldest a farmer who lived 7,000 years ago in what is now Germany. Until now, the oldest viral DNA ever recovered from human remains was just 450 years old.” Here is the abstract from the article: 

“The hepatitis B virus (HBV) is one of the most widespread human pathogens known today,  yet its origin and evolutionary history are still unclear and controversial. Here, we report the analysis of three ancient HBV genomes recovered from human skeletons found at three  different archaeological sites in Germany. We reconstructed two Neolithic and one medieval  HBV genomes by de novo assembly from shotgun DNA sequencing data. Additionally, we observed HBV-specific peptides using paleo-proteomics. Our results show that HBV circulates in the European population for at least 7000 years. **The Neolithic HBV genomes show a high genomic similarity to each other. In a phylogenetic network, they do not group with any human-associated HBV genome and are most closely related to those infecting African non-human primates. These ancient virus forms appear to represent distinct lineages that have no close relatives today and went possibly extinct.** Our results reveal the great  potential of ancient DNA from human skeletons in order to study the long-time evolution of blood borne viruses.”

Note the highlighted lines from the abstract.

Figure-2: Network of 495 modern, two published ancient genomes (light yellow box), and three ancient hepatitis B virus (HBV) obtained in this study (grey box). Colors indicating the 8 human HBV Genotypes (A-H), two monkey genotypes (Monkeys I, African apes and Monkeys II, Asian monkeys) and ancient genomes (red).[research article](https://www.biorxiv.org/content/early/2018/05/06/315531.full.pdf+html)


**Q.6. What is the role of evolution (as an algorithm) in this example?**

**Q.7. Thinking back to week-3, DNA read/write, what is the role of synthesis and sequencing in realizing this example?**


## Summary

Through these examples, you have started to synthesize the workings of evolution and natural selection into a streamlined algorithm. This week you will engage with this algorithm as a tool (and service) that can be used to solve challenges and problems in science/engineering and perhaps beyond?). In doing so, you will start to identify the current limitations of this deployment of evolution, and consider what it would mean to “improve” evolution. 


*Additional Resources:*
 Please make sure to briefly take a look at these great resources: 
 
 [1. Collection of links - excellent place to learn about fundamental concepts in evolution](https://www.nature.com/scitable/knowledge/library/evolution-is-change-in-the-inherited-traits-15164254) 
 
 [2. Misconceptions about evolution](https://evolution.berkeley.edu/evolibrary/misconceptions_teacherfaq.php#a2)
 
 [3. Host and Viral Evolution: Molecular Evolutionary Arms Race Between Primate and Viral Genomes link](https://www.ibiology.org/microbiology/viral-evolution/)

[4. Spanish Influenza Pandemic and Vaccines](https://www.historyofvaccines.org/content/blog/spanish-influenza-pandemic-and-vaccines)



_______________________________________________________________________

## Preclass for Wedensday 


## Introduction:From Natural Algorithm to Engineering Service - part 2

In the first class you started to learn about evolution and its  role from a Bioengineering perspective.  In the previous class you were introduced to evolution as an algorithm. In this class you will examine the role of evolution as an engineering solution and approach. After considering the following example and the material throughout this week you will be able to:

**Goal-1:** Abstract evolution from a population-wide biological force to a service that can be deployed by a group or an individual. (“Evolution, at your service!”) 

**Goal-2:** Be able to identify when evolution is a good engineering choice and why? (also begin to address who gets to decide?) 

**Goal-3:** Understand the limitations and benefits of this approach.  


 ## Example-1: “Evolution, at your service!” 
 
Since the days of Gregor Mendel, geneticists (and avid gardeners/monks) have understood that we can breed populations for certain traits.  By taking advantage of our understanding of heritability and genetics, we can - for example - make chickens with extravagant traits ([link](https://www.pinterest.co.uk/pin/153685406008776095/), [link](https://www.pinterest.com/pin/266416134178755168/))

Figure-1: goes here


Another example of human intervention in evolution is dog breeding. Take a look at the link to see the evolution of Petface [here](https://www.smithsonianmag.com/science-nature/evolution-petface-180967987/) From the article: Cranial length, snout length and skull length all contribute to the difficulties brachy dogs have in breathing. (PLOS One)

**Q.1. Think back to the pre-class material for monday. What are the selection pressures (and their sources) in the case of flu and dog/chicken breeding? How is the fitness determined in each case?**

**Q.2. Thinking of evolution as an algorithm, where else can we intervene?** 


 ## Example-2: Directed evolution of proteins 
 
[Directed evolution](https://www.nobelprize.org/prizes/chemistry/2018/press-release/) is an effective strategy for improving or altering the activity of biomolecules for industrial, research and therapeutic applications. Laboratory evolution of proteins requires methods for generating genetic diversity coupled with identifying protein variants with desired properties. 

Here is an introduction from a [research article](https://www.nature.com/articles/nrg3927)

“ Over many generations, iterated mutation and natural selection during biological evolution provide solutions for challenges that organisms face in the natural world. However, the traits that result from natural selection only occasionally overlap with features of organisms and biomolecules that are sought by humans. To guide evolution to access useful phenotypes more frequently, humans for centuries have used artificial selection, beginning with the selective breeding of crops and domestication of animals. More recently, directed evolution in the laboratory has proved to be a highly effective and broadly applicable framework for optimizing or altering the activities of individual genes and gene products, which are the fundamental units of biology Genetic diversity fuels both natural and laboratory evolution. The occurrence rate of spontaneous mutations is generally insufficient to access desired gene variants on a time scale that is practical for laboratory evolution. A number of genetic diversification techniques are therefore used to generate libraries of gene variants that accelerate the exploration of a gene's sequence space. Methods to identify and isolate library members with desired properties are a second crucial component of laboratory evolution. During organismal evolution, phenotype and genotype are intrinsically coupled within each organism. However, during laboratory evolution (Figure-3), it is often inconvenient or impossible to manipulate genes and gene products in a coupled manner. Therefore, single-gene evolution in the laboratory requires carefully designed strategies for screening or selecting functional variants in ways that maintain the genotype–phenotype association.

**Figure-3:** The process of directed evolution in the laboratory mimics that of biological evolution.  A diverse library of genes is translated into a corresponding library of gene products and screened or selected for functional variants in a manner that maintains the correspondence between genotype (genes) and phenotype (gene products and their functions). These functional genes are replicated and serve as starting points for subsequent rounds of diversification and screening or selection. B. Although the mutational space is multidimensional, it is conceptually helpful to visualize directed 
evolution as a series of steps within a three-dimensional fitness landscape. Library generation samples the proximal surface of the landscape, and screening or selection identifies the genetic means to 'climb' towards fitness peaks. Directed evolution can arrive at absolute maximum activity levels but can also become trapped at local fitness maxima in which library diversification is insufficient to cross 'fitness valleys' and access neighbouring fitness peaks [Source](https://www.nature.com/articles/nrg3927).

**Natural selection:** A process by which individuals with the highest reproductive fitness pass on their genetic material to their offspring, thus maintaining and enriching heritable traits that are adaptive to the natural environment. 

**Artificial selection:** A process by which human intervention in the reproductive cycle imposes a selection pressure for phenotypic traits desired by the breeder. (Also known as selective breeding).  

**Libraries:** Diverse populations of DNA fragments that are subject to downstream screening and selection. 

**Q.3. How can directed evolution be used an approach to evolve desired proteins?**

**Q.4. What are the challenges associated with this approach?** 

**Q.5. When is directed evolution a good choice?**


 ## Example-3: Antibiotic resistance
 
 Antibiotic resistance and the threat of drug resistance bacteria is an emerging challenge that grows every year.
 ([link-1](https://www.theatlantic.com/science/archive/2017/02/in-bacteria-persistence-leads-to-resistance/516138/), 
 [link-2](https://www.theatlantic.com/science/archive/2016/05/the-ten-part-plan-to-avert-our-post-antibiotic-apocalypse/483360/). 
 

According to [CDC](https://www.cdc.gov/antibiotic-use/community/about/antibiotic-resistance-faqs.html):
 
“Overuse and misuse of antibiotics can promote the development of antibiotic resistance. Every time a person takes antibiotics, sensitive bacteria (bacteria that antibiotics can still attack) are killed, but resistant bacteria are left to grow and multiply. This is how repeated use of antibiotics can increase the number of drug-resistant bacteria.  Antibiotics are not effective against viral infections like the common cold, flu, most sore throats, bronchitis, and many sinus and ear infections. Widespread use of antibiotics for these illnesses is an example of how overuse of antibiotics can promote the spread of antibiotic resistance. Smart use of antibiotics is key to controlling the spread of resistance.”
 
 
**Q.6. What is the role of evolution in antibiotic resistance?**

To answer the question above recall that in an earlier module you came across the MEGA-plate. 
The (120 × 60 centimeters) plate on which bacteria spread and evolved on a large antibiotic landscape
([Mega-plate video](https://www.youtube.com/watch?v=plVk4NVIUh8)

Here is the abstract from the [paper](http://science.sciencemag.org/content/353/6304/1147.full):

"A key aspect of bacterial survival is the ability to evolve while migrating across spatially varying environmental challenges. Laboratory experiments, however, often study evolution in well-mixed systems. Here, we introduce an experimental device, the microbial evolution and growth arena (MEGA)–plate, in which bacteria spread and evolved on a large antibiotic landscape (120 × 60 centimeters) that allowed visual observation of mutation and selection in a migrating bacterial front. While resistance increased consistently, multiple coexisting lineages diversified both phenotypically and genotypically. Analyzing mutants at and behind the propagating front, we found that evolution is not always led by the most resistant mutants; highly resistant mutants may be trapped behind more sensitive lineages. The MEGA-plate provides a versatile platform for studying microbial adaption and directly visualizing evolutionary dynamics." 

**Q.7. Could we use evolution to combat antibiotic resistance?**


*Useful Additional Resource:* 
Review the key Takeaways from: 
[Methods for the directed evolution of proteins](https://www.nature.com/articles/nrg3927)

## Preclass Reading for Fridy  

### Meet Your Microbes!

After finishing this activity:

**Goal-1:** You will have been introduced to basic methods for cultivating andcategorizing 
microbial life in the lab; it turns out we can only cultivate ~1% of microbes found in nature in the lab 
(i.e., we can’t grow most microbes underlaboratory conditions, at least not yet).

**Goal-2:** You have been introduced to the diversity of the life (microbiome) that
live in and on you (are you?). And, begin to describe the role that the microbiome plays in your health and well being.

**Goal-3:** By observing this diversity of microbial phenotypes you will have theopportunity to appreciate the rich diversity and engineering potential that livingmatter provides.

<figure>
<a href="/assets/images/w4pc_S.Chimileski Biofilm_pseudomonas aeruginosa biofilm.gif"><img src="/assets/images/w4pc_S.Chimileski Biofilm_pseudomonas aeruginosa biofilm.gif"></a>
</figure>


**Figure-1** biofilm development via a *Pseudomonas aeruginosa* colony with the extracellularmatrix, material released by the cells, stained red and pink by Scott Chimileski. [Source](https://www.scottchimileskiphotography.com/Creative-Commons/) - licensed under a Creative Commons non-commercial use. Make sure to visit his [website](https://www.scottchimileskiphotography.com/Portfolio/) for additional photos.

In an earlier activity we used a Foldscope to observe and explore the world around us,either as explorers or makers looking for parts and inspiration. In today’s activity we willexplore the *microbiome* that lives on and within us.
The microbiome is the vast collection of bacteria, viruses, fungi, and other microbes thatshare our bodies (additional [link](https://www.nature.com/collections/stkpgwjvvk/content/content). Depending on their context they can play a positive
or a negative role in our health and well being. For example, they can cause orinfluence non-communicable diseases. Engineering the microbiome is considered by some respected academics to be part of the the future of medicine. For example,researchers at Cornell recently engineered a gut microbe to address aspects of diabetes 
(see [here](https://www.ncbi.nlm.nih.gov/pubmed/25626737)).

**Q.1. What do you think are various methods that we can use to study themicrobiome (i.e., all bacteria in and on you)? How can we tell what microbes arepresent in the microbiome?**

Thinking back to observing via your Foldscope, looking at microbes is certainly onemethod to study microbes. 
Figure-2 (below) are photographs of multiplebacteria and fungal colonies. You may notice the different colors, shapes, sizes, andstructures of these colonies. Some of the colonies have a smooth morphology whileothers have a more folded surface. You may begin to ask what are the advantages of having a smooth, or folded, or a rough surface. You may also use this diversity of properties as a possible approach to discern various microorganisms from one another.  However there is one important caveat to keep in mind, the diversity of microbial worldis incredible but we can only cultivate for study approximately 1% of the microbes thatlive on you. This is in part because many microbes are highly “social” and dwell in andamong communities of other microbes. We still have much to learn about the 1% we can cultivate, with plenty more to learn from the 99% we can’t yet grow.

<figure>
<a href="/assets/images/w4pc_meetYourMicrobe.png"><img src="/assets/images/w4pc_meetYourMicrobe.png"></a>
</figure>

**Figure-2**  Photographs of multiple bacteria and fungal colonies.  It is possible to identify bacterial or fungal species by the shape, color, and structure of the colonies that they form. 


**Q.2. If we cannot cultivate and grow many components of the microbiome inisolation in the lab, how can we tell what is present in each context? Whatcontemporary technologies have enabled the study and engineering of themicrobiome?**

Think about your own answer and then visit the following link to begin to answer the above question [Scientists Unveil New ‘Tree of Life’](https://www.nytimes.com/2016/04/12/science/scientists-unveil-new-tree-of-life.html)


While the plate assays (growing bacteria on defined media) and looking at growth andphenotypes, may seem simple. They have and continue to play an incredibly powerfulrole in the discovery and development of biology as well as in the pursuit of bioengineering.[source](http://science.sciencemag.org/content/353/6304/1147) demonstrates a creative and unique plateassay called MEGA-plate (“mega” since the plate is **2x4 feet!**) to demonstrate and study bacterial evolution. MEGA plate was used to study the evolution andemergence of antibiotic resistance over time in bacterial populations. Make sure to watch this [video](https://www.youtube.com/watch?time_continue=7&v=yybsSqcB7mE).


### What Next
During the class, you will use the plates that you prepared on Wednesday toexamine and classifying the microbes 
that grew on your plates. To prepare for this make sure to take a look at the first page of this 
[link](http://delrio.dcccd.edu/jreynolds/microbiology/2421/lab_manual/colony_morph.pdf)
