---
title: "Week 6 Preclass"
permalink: /docs/w6preclass/
last_modified_at: Feb.20.2019
toc: true
---

## What arises from DNA read/write trends that will matter for the rest of our lives?

## Preclass for Monday
Please read and consider the below before start of class on Monday.
The questions given are only study questions not homework to be graded.
Talk about it all with your classmates, friends, or TAs, as you like.


## Introduction |Technology, Tools, and Trends (DNA read and write)

This week we will explore **What changes may arise via trends in DNA read and write tools may matter for the rest of our lives?**  We hope you will learn more about the core capacities for accessing and encoding DNA and also consider the quantitative pacing of how these tools are improving. Monday's pre-class material focuses mostly on DNA sequencing, whereas Wednesday's pre-class material focuses on DNA synthesis...

**Goal 1:** You should be able to describe how DNA sequencing works.

**Goal 2:** You should be able to describe the pace of developments in DNA read and write and be ready to discuss the ramifications of this development.

**Goal 3:** You should also be able to describe how DNA synthesis works.

**Goal 4:** You should gain a capacity to strategically evaluate quantitative trends and qualitative transitions.

Let’s begin!

## Example 1: Non-invasive prenatal testing

Put yourself in the shoes of a couple who recently discovered they were pregnant, andare worried about genetic disorders in their unborn child (e.g., Down syndrome).

**Scenario-1:** It is 2010 and the standard of care is amniocentesis, an invasive procedurein which a large needle is used to directly sample fluid from the amniotic sac [optional link](https://www.youtube.com/watch?v=XXIW8GHwJ_M)

**Q.1. What do you think about the test, would you take it? Why?**

**Scenario-2:** It is 2018 and in leading-edge children’s hospitals amniocentesis has beenreplaced by non-invasive prenatal testing (NIPT), previously known as cell free DNA testing. NIPT is based on a simple blood draw that enables determination of geneticdisorders and even the sex of an unborn fetus. The test works because it turns out that there is DNA from the fetus circulating in the pregnant woman’s blood (e.g., if apregnant woman has Y chromosome DNA in her blood then she is carrying a malefetus). Please watch [video](https://www.youtube.com/watch?v=e86_l_UqecU) about cell free fetal DNA analysis if you want moredetail.


<figure>
<a href="/assets/images/w3pc_CarlsonCurves.png"><img src="/assets/images/w3pc_CarlsonCurves.png"></a>
</figure>

**Figure.1.** "Reduction of the Price of DNA sequencing (reading) and synthesis(writing) per base per over past 30 years.  [Source](https://synbiobeta.com/time-new-dna-synthesis-sequencing-cost-curves-rob-carlson/)

**Q.2. Do you think NIPT could have been economically feasible in the year 2010?What about 2000, or 1990? Why?**

**Q.3. What factors do you think led to exponential cost reduction in DNA sequencing?**

## Example 2: DNA literacy (DNA read)

DNA is made up of nucleotides. These nucleotideshave four possible bases;
adenine (A), thymine (T),guanine (G), and cytosine (C). All DNA is made up of sequences of these four bases. Imagine a four key typewriter that is able to type the DNA codes for all of the biodiversity seen on earth. Being able to both **read DNA** (sequencing) and **write DNA** (synthesizing) has and will continue to givebioengineers the ability to design and build with livingmatter in ways that were hard to imagine very recently.

Let’s delve into how current sequencing technologies work.

Start with this [video](https://www.youtube.com/watch?v=MvuYATh7Y74) that provides backgroundabout sequencing and its use to decode the human genome, with a focus on “massivelyparallel sequencing” (often marketed as “next-generation” sequencing technology). Figure-2, also breaks down the same process described in the video.

<figure>
<a href="/assets/images/w3pc_NGS.png"><img src="/assets/images/w3pc_NGS.png"></a>
</figure>

**Figure.2.** "Breakdown of DNA sequencing steps (specifically next generationsequencing). In this approach instead of reading a long sequence of DNA one baseper at a time, the sequence is divided into many pieces (fragments) which are all read simultaneously". [Source](https://www.nature.com/articles/nature24286.pdf)

During sequencing first, the genomic DNA (the DNA that we want to read) is broken into smaller pieces (fragmented). Then a synthetic linker sequence is added to the end ofthe fragments (adaptor) which helps the DNA fragments get attached to a surface on which the sequencing happens (flow cell). Next, the DNA fragments are separated inspace and read individually. Each fragment is amplified (thousands of copies of eachpiece is made) to make sure the correct sequence is detected. To actually read the fragments, pre-labeled nucleotides (nucleotides with color) and enzymes (that incorporate nucleotides into the fragments) are added to the flow cell. As each colored base pair is added to the fragments a new colorful spot is introduced. By taking images from the spots we are able to decode the sequence. Finally, the fragments are stitched together to reveal the original sequence of the genomic DNA. Here is an additional video that describes this process in even more detail [optional link](https://www.youtube.com/watch?v=fCd6B5HRaZ8).

**Q.4. According to the “How to sequence the human genome” video when was thefirst human genome sequenced? How long did it take? How much did it cost?**

**Q.5. How much would it cost if you were to sequence the human genome in 2018?**

**Q.6. Thinking back to the design, build, and test (DBT) class. What are theconsequences of lowering sequencing cost (both money and time)?**

**RESOURCES**

If the links above do not work and you need to type them out, here are the full URLs foreach link:

1. Amniocentesis procedure video: [link](https://www.youtube.com/watch?v=XXIW8GHwJ_M)

2. Cell free fetal DNA analysis video: [link](https://www.youtube.com/watch?v=8Kynt75ssmE)

3. Sequencing video: [link](https://www.youtube.com/watch?v=MvuYATh7Y74)

Image Sources:
1. Carlson Curves [link](https://synbiobeta.com/time-new-dna-synthesis-sequencing-cost-curves-rob-carlson/)
2. Next-Generation Sequencing: Shendure et al, Nature 2017  [link](https://www.nature.com/articles/nature24286.pdf)

____________________________________________________________

## Preclass for Wednesday
Please read and consider the below before start of class on Wednesday

**DNA Synthesis**

1. A less than 4 minute video introducing how Agilent builds 60mer oligos using inkjet dispensing.
[link](https://www.youtube.com/watch?v=eSr5CxAdiww&feature=youtu.be)

2. A summary of existing and recent DNA synthesis companies:
[link](http://www.calinplesa.com/dna-synthesis-companies/)

3. A recent article on difference approaches to DNA synthesis:
[link](https://www.chemistryworld.com/feature/step-by-step-synthesis-of-dna/3008753.article)

If you are short on time just watch the video (i.e., please consider (2) and (3) as nice optional/supplements).
____________________________________________________________

## Preclass Reading for Friday  

### Meet Your Microbes!

After finishing this activity:

**Goal-1:** You will have been introduced to basic methods for cultivating andcategorizing
microbial life in the lab; it turns out we can only cultivate ~1% of microbes found in nature in the lab
(i.e., we can’t grow most microbes underlaboratory conditions, at least not yet).

**Goal-2:** You have been introduced to the diversity of the life (microbiome) that
live in and on you (are you?). And, begin to describe the role that the microbiome plays in your health and well being.

**Goal-3:** By observing this diversity of microbial phenotypes you will have theopportunity to appreciate the rich diversity and engineering potential that livingmatter provides.

<figure>
<a href="/assets/images/w4pc_S.Chimileski Biofilm_pseudomonas aeruginosa biofilm.gif"><img src="/assets/images/w4pc_S.Chimileski Biofilm_pseudomonas aeruginosa biofilm.gif"></a>
</figure>


**Figure-1** biofilm development via a *Pseudomonas aeruginosa* colony with the extracellularmatrix, material released by the cells, stained red and pink by Scott Chimileski. [Source](https://www.scottchimileskiphotography.com/Creative-Commons/) - licensed under a Creative Commons non-commercial use. Make sure to visit his [website](https://www.scottchimileskiphotography.com/Portfolio/) for additional photos.

In an earlier activity we used a Foldscope to observe and explore the world around us,either as explorers or makers looking for parts and inspiration. In today’s activity we willexplore the *microbiome* that lives on and within us.
The microbiome is the vast collection of bacteria, viruses, fungi, and other microbes thatshare our bodies (additional [link](https://www.nature.com/collections/stkpgwjvvk/content/content). Depending on their context they can play a positive
or a negative role in our health and well being. For example, they can cause orinfluence non-communicable diseases. Engineering the microbiome is considered by some respected academics to be part of the the future of medicine. For example,researchers at Cornell recently engineered a gut microbe to address aspects of diabetes
(see [here](https://www.ncbi.nlm.nih.gov/pubmed/25626737)).

**Q.1. What do you think are various methods that we can use to study themicrobiome (i.e., all bacteria in and on you)? How can we tell what microbes arepresent in the microbiome?**

Thinking back to observing via your Foldscope, looking at microbes is certainly onemethod to study microbes.
Figure-2 (below) are photographs of multiplebacteria and fungal colonies. You may notice the different colors, shapes, sizes, andstructures of these colonies. Some of the colonies have a smooth morphology whileothers have a more folded surface. You may begin to ask what are the advantages of having a smooth, or folded, or a rough surface. You may also use this diversity of properties as a possible approach to discern various microorganisms from one another.  However there is one important caveat to keep in mind, the diversity of microbial worldis incredible but we can only cultivate for study approximately 1% of the microbes thatlive on you. This is in part because many microbes are highly “social” and dwell in andamong communities of other microbes. We still have much to learn about the 1% we can cultivate, with plenty more to learn from the 99% we can’t yet grow.

<figure>
<a href="/assets/images/w4pc_meetYourMicrobe.png"><img src="/assets/images/w4pc_meetYourMicrobe.png"></a>
</figure>

**Figure-2**  Photographs of multiple bacteria and fungal colonies.  It is possible to identify bacterial or fungal species by the shape, color, and structure of the colonies that they form.


**Q.2. If we cannot cultivate and grow many components of the microbiome inisolation in the lab, how can we tell what is present in each context? Whatcontemporary technologies have enabled the study and engineering of themicrobiome?**

Think about your own answer and then visit the following link to begin to answer the above question [Scientists Unveil New ‘Tree of Life’](https://www.nytimes.com/2016/04/12/science/scientists-unveil-new-tree-of-life.html)


While the plate assays (growing bacteria on defined media) and looking at growth andphenotypes, may seem simple. They have and continue to play an incredibly powerfulrole in the discovery and development of biology as well as in the pursuit of bioengineering.[source](http://science.sciencemag.org/content/353/6304/1147) demonstrates a creative and unique plateassay called MEGA-plate (“mega” since the plate is **2x4 feet!**) to demonstrate and study bacterial evolution. MEGA plate was used to study the evolution andemergence of antibiotic resistance over time in bacterial populations. Make sure to watch this [video](https://www.youtube.com/watch?time_continue=7&v=yybsSqcB7mE).


### What Next
During the class, you will use the plates that you prepared on Wednesday to examine and classifying the microbes
that grew on your plates. To prepare for this make sure to take a look at the first page of this
[link](http://delrio.dcccd.edu/jreynolds/microbiology/2421/lab_manual/colony_morph.pdf)
