---
title: "Week 6 PSET"
permalink: /docs/w6ps/
2018:
---

**DUE:** Next Friday (May 17) beginning of class.

**NOTES:**
  - All BIOE.80 problem sets must be completed individually unless plainly noted otherwise.
  - Please turn in your completed problem sets as an electronic copy via Gradescope.
  - Please make sure to not go over the word limits and when appropriate show your work (e.g., calculations) and include units.

**GOAL:** Welcome to PSET-6. This problem set will help you further develop your ability to
describe how DNA read and write work. Coupled with this you will examine the consequences of
non-trivial *quantitative pacing* in the development of these technologies. Finally, you will examine the consequence of  interconvertability of matter and information as DNA read and write tools improve and discuss the *qualitative changes* that emerge.


## (Q1) (Nature + Nurture) or (Fab a Family) (20 pts)

Over the past 12 years the price of synthesizing genes has dropped from $4 to $0.04
per base pair; presume the future price of DNA synthesis will continue to drop two fold every two years.

Meanwhile Stanford’s undergraduate tuition is approximately $50,000 per year up from $25,000 in 2000.  
Presume Stanford’s tuition will continue to double every 15 years.

**Q.1.a.** If a human genome is 4 billion base pairs long then when will the cost of synthesizing the DNA encoding an entire human genome be roughly the same as the tuition cost associated with attending Stanford for one year?

Hint: Use the facts given. Keep your math simple, and write out the equations (i.e., show your work).
An approximate answer is fine.

**Q.1.b.**  What would be your estimate if the price of synthesizing genes continue to drop two fold every 2.5 years instead of 2 years (i.e., only a 6-month longer doubling time). An approximate answer is fine.

**Q.1.c.** Given your answer from part (a) presume that you could then design, build, and bring to life a human encoded by a genome that you design (e.g., with any and all mutations that you and your partner choose).  As potential parents would you prefer to spend your savings on college tuition for a natural child or on realizing a genetically engineered offspring who then has to make  their own way? Why? (2-3 sentences)

## (Q2) DNA as a Storage Medium (30 pts)

The US government has been using an underground salt mine in Pennsylvania as a long term storage facility for physical documents, primarily retirement related papers. According to the Washington Post, in this facility approximately 1000 employees pass thousands of case files from cavern to cavern and then type in retirees’ personal data, one line at a time. Basic document retrieval involves a forklift and takes days. Given the trends in DNA read and write, consider using DNA as an alternative storage medium for these documents.

Helpful numbers:

Volume of storage facility = 10 million m^3

Letters per cubic meter of typical government document = 10^9 letters/ m^3

Volume of a DNA base pair ~ 10^-27 m^3

Please round or simplify whenever convenient. Please show your work and include units.

**Q.2.a.** Since DNA has only 4 bases, but English has 26 letters, you will somehow have to map letters to bases. Briefly specify a simple encoding scheme for doing that. You *do not* have to give a letter to DNA mapping for all letters.
The first few will do the trick.

**Q.2.b.** Using your letter to DNA mapping, how many total base pairs of DNA would be needed to store the archival contents of the salt mine. Assume that the salt mine is completely packed with documents.

**Q.2.c.** What would be the physical volume of this DNA?

**Q.2.d.** Will the DNA fit into a thimble?  Yes / No

**Q.2.e.**  How expensive would it be to print this DNA (with cost ~1 cents/base)

**Q.2.f.** Give two major advantages of using DNA as storage medium (bullet points)

**Q.2.g.**  Give two major disadvantages of using DNA as storage medium (bullet points)

**Q.2.h.** How would you maintain and copy (replicate) this library (bullet points)

**Q.2.i.** Can information be stored in the DNA in a living organism as opposed to DNA stored in a frozen tube (-80 freezer).
If yes, what are one advantage and one disadvantage of using live cell storage? (2 bullet points)

Helpful resources:[link1](https://www.nature.com/news/how-dna-could-store-all-the-world-s-data-1.20496), [link2](https://wyss.harvard.edu/taking-cells-out-to-the-movies-with-new-crispr-technology/)

## (Q3) Sequencing the Total DNA in the Biosphere (10 pts)

**Q.3.** The following [paper](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002168) estimates that there are about 5.3 x 10^37 bases of DNA in the biosphere.  Given the increasing trends in productivity of sequencing (see slide-2 from class on Wednesday). In what century can a team of 10 sequence 5.3 x 10^37 bases in a month?

![Carlson Curves](/assets/images/PSET6_Crlson_Productivity.png
 "Productivity in DNA Sequencing and Synthesis")

**Figure-1:** Carlson Curves.
 
## (Q4) Meet your Microbes (40 pts)

On Wednesday and Friday of this week, you cultivated, grew, and examined the microbes from your skin.

**Q.4.a.** Did you show up on Friday? (yes/no)

**Q.4.b.** Define, in your own words, a microbiome (1-2 sentences).

**Q.4.c.** **Colony Morphology** Examine the morphology of the colonies on a plate from your table (your plate or a neighbor's  plate).
Describe as comprehensively as possible the ONE most interesting colony at your table (you decide what’s interesting!).
What is the size, shape, and color of the colony? What is the narrowest classification you can make of this colony (or, in other words, what type of bacteria do you think this might be)?  (4-5 sentences; make sure to include a picture and figure caption!)

**Sequencing microbes**

As sequencing has become cheaper and more accessible, we can now generate huge datasets of sequences. Because of this, we no longer have to rely on culturing microorganisms (ie: growing them on a plate in the lab) to know what’s there. We can take a sample, break open the cells to extract DNA and sequence all of the DNA that is present.  

In [this paper](https://www.nature.com/articles/nature13786), a group sequenced several skin samples from the human body. Below, you can see a table of the top 15 most abundant bacteria (classified at the genus level) for one healthy individual’s left palm, given as a % relative abundance (ie: 100 would be a pure culture of 1 single organism). The relative abundance of a sample from that same individual’s right palm is also given. 

**Table 1:** Relative Abundance of Microbes on an Individual's Left and Right Palm

![Table of Relative Abundance](/assets/images/Microbes_PalmSample_Table.png
 "Relative Abundance of Microbes on Individual's Left and Right Palm")
 
**Q.4.d.** Take a look at this data and compare it to what you saw on your plates. Write down two observations that you find interesting about this data. 

**Q.4.e.** In class, we mentioned four genera that are likely to be found on your plates: *Staphylococcus*, *Corynebacterium*, *Acinetobacter*, and *Pseudomonas*. What is the combined relative abundance of these types of bacteria for this individual’s left palm? Is this number surprising to you? Why or why not?

**Q.4.f.** What are two advantages to using a sequencing approach to identify microbes? What are two advantages to culturing organisms? 

**Q.4.e** Look up one of these microbes from the table. Include a picture and a short paragraph (3-4 sentences) about what is known about this microbe. A picture can take the form of an image of a colony on a plate or a microscope-based image, including a SEM. Make sure to include your source(s).

**Additional Resources for microbial sequencing of skin microbiome:**

When dealing with microbes, a common strategy to identify what microbes are present is to look at a biomarker on their DNA that is present in all bacteria. For this to work, the biomarker must contain regions that are highly similar to each other as well as variable regions that can be used to identify specific types of bacteria.  For an overview of the most commonly used taxonomic marker, a sequence from the small subunit of the ribosome (16S rRNA), see Wikipedia: https://en.wikipedia.org/wiki/16S_ribosomal_RNA 

Here are two reviews about the skin microbiome:
[Link 1](https://www.nature.com/articles/nrmicro.2017.157)

[Link 2](https://www.nature.com/articles/nrmicro2537)

Feel free to reach out if you’re interested in the human microbiome or other microbiomes in general! We’d be happy to direct you to additional resources.

## Final Notes 

This week you learned about tools and technologies that enable DNA read and write. As part of your core bioengineering skills you should be able to describe how these technologies work. You also learned about the non-trivial speed of development in these technologies and the various ramifications of this development.

**Do you have any unanswered questions (from this week, past weeks, or general BIOE) that you would like to share with us?**


## Extra learning (100% optional):

### (Q5)Sequencing and Synthesis technologies (0 pts)

In the pre-class material you were introduced to a DNA read method commonly referred to as next-generation sequencing (NGS) or massively parallel sequencing. Let’s examine an alternative approach, namely nanopore sequencing.


Watch this [video](https://nanoporetech.com/products/minion)

**5.a.** Describe how Nanopore sequencing works (use bullet points)

MinION is a product that uses Nanopores for sequencing.
While the error rate with Nanopore technology is between 5-10%. [additional Link](https://f1000research.com/articles/6-760/v1). This tool can be used to sequence and assemble a human genome
[additional link](https://www.nature.com/articles/nbt.4060).  Now briefly examine the various products sequencing technology from Illumina [link](https://www.illumina.com/systems/sequencing-platforms/comparison-tool.html).

**5.b.** How are NGS and Nanopore sequencing different?

**5.c.** Imagine that the storage of information in DNA from the previous question has become a routine platform.
You have been hired as a consultant for a library that seeks to store its archive inside the DNA medium.
Your task is to determine suitable reading platforms. What type of“read technology” would you recommend:
a sequencer that gives massive simultaneous short reads with very low error,
or fast readers that reads long sequences with error rates as high as 15 %.
Please include quantitative reasoning in your answer.  

**5.d.** Where would you use an Illumina sequencing machine instead of a minION sequencing machine?
and vice versa?
