---
title: "Week 8 Problem setslass"
permalink: /docs/w4ps/
---

# Problem Set Week-8

**DUE:** Next Friday start of class via Canvas.

**NOTE:** All BIOE.80 problem sets must be completed individually unless plainly noted otherwise.

**NOTE:** Please turn in your completed problem sets as an electronic copy via Canvas.

**NOTE:** Please make sure to not go over the word limits.

**Goals:** Welcome to PSET-4. 
In this PSET you will review the minimal genome paper as practiceto learn how to read research papers. 
You will also improve your quantitative intuition with anestimation problem, 
and finally revisit the meet your microbe activity.

## (Q1) Design and Synthesis of a Minimal Bacterial Genome (40 pts)

A key skill that you will need as bioengineers is to read, understand, and extract informationfrom research papers that are shaping and defining the frontiers of bioengineering. This is inpart due to the fast changing pace of bioengineering. We would like you to practice this skill as early as possible, to help you become better at it.


In class we started to discuss the paper *“Design and synthesis of a minimal bacterial genome"*. In this problem you will review the the paper to level up your ability to read and extractinformation from a research paper.

1.a. Re-read the abstract from the paper, in your own words describe what was the paper’s goal(s), 
what did the authors discover by working on these goal(s). (bullet points)

1.b. Take a look at Figure-1 from the paper titled: “The JCVI DBT cycle for bacterial genomes”. 
Think back to what you have learned during the DBT Week. Based on the figure and your prior knowledge, 
what tools and technologies were used to design, build, and test Syn3.0.?  (bullet points)

1.c. Take a look at Figure-2 titled: “Strategy for whole-genome synthesis”. What do the red, blue, and green arrows represent? What does the orange arrow represent? 
Using the figure briefly explain how they went from Oligos to a whole genome? (bullet points)

1.d. Assuming $0.04 per base pair what is the minimum cost ofsynthesizing the Syn3.0 genome?

1.e. Take a look at Figure-7. Let’s compare the phenotype of Syn1.0 and Syn3.0.
Describe what you see when comparing panel A and B and panel C?(bullet points) 
Which cellsgrow faster? ( Syn1.0 or Syn3.0.)

## (Q2) Approximation and estimation - Genome Size (40 pts)

Now thatyou have examined a minimal genome have you wondered how big is a typical genome?

The *E. coli* MG1655 genome is 4.6 Mbp (megabasepairs) long, or approximately 4,600,000base pairs. 
A good rule of thumb for the length of a single DNA base pair is that it is ⅓ nm long.
As another rule of thumb, let’s treat the volume of an *E. coli* cell as being approximately 1 μm^3 [see](http://book.bionumbers.org/how-big-is-an-e-coli-cell-and-what-is-its-mass/). This is a rough, order-of-magnitude estimate. 

In real life, cell size and volume willv ary based upon species, growth rate, and stage of division 
in addition to many other factors.

2.a. How long would the *E. coli* genome be as a linear strand of DNA?  Provide your answerin μm. How long would the Syn3.0 genome be as as a linear strand of DNA?

2.b. The *E. coli* genome is actually circular. What is the radius of the genome, assuming thegenome is arranged as a perfect circle and given the linear length that you calculated above?

2.c. Consider your answers in relation to the size specifications for *E. coli* given above. Whatdoes this imply about the layout of DNA inside a living cell? (Two sentences).

Note: A good rule of thumb for the volume of a DNA base pair is that one base pair has avolume of approximately 1 nm^3.

2.d. How much DNA could be packed into an *E. coli* cell, assuming that the whole cellvolume only contains DNA? Why is this number ridiculous? (i.e., the *E. coli* genome issignificantly smaller: why?)

Additional resources: If you like to learn more about various genome sizes please visit this [link](http://book.bionumbers.org/how-big-are-genomes/)


## (Q3) Meet your Microbe (20 pts)

On Wednesday and Friday of this week you cultivated, grew, and examined the microbes from your skin.

3.a. Describe in your own words what is the microbiome, and provide a current example ofstudy of the microbiome (include references if needed) (Max 100 words)

3.b. What are key methods used to study the microbiome? (Bullet points)

3.c. What are the current and future roles of DNA sequencing and synthesis in thestudy and engineering of microbiome? (1 or 2 Bullet points for each)

3.d. **Colony Morphology** Examine the morphology of the colonies on your plates. Describe as comprehensively as possible the ONE most interesting colony at your table (you decide what’s interesting!). What is the size, shape, and shape of the colony? What is thenarrowest classification you can make of this colony? Make sure to include a picture. (Max 100 words)

**Final Notes:** The Ability to read (and try to understand) primary sources (research papers) is a core skill that you will need both as a student and beyond college. This week you learned about Syn3.0 and how through building a minimal genome researchers discovered 149 essential genes that were previously unidentified. The paper highlighted our improved ability to synthesize genes, and combine them together to create a minimal genome. At the same time it highlighted our critical gaps in knowing what genes might be essential ahead of time.

**Your questions**
Do you have any unanswered questions (from this week, past weeks, or general BIOE) that you would like to share with us?
